crates/aptos/src/move_tool/mod.rs

```diff
@@ -1342,6 +1342,13 @@ impl FunctionArgType {
                         hex::decode(arg)
                             .map_err(|err| CliError::UnableToParse("vector<hex>", err.to_string()))
                     }),
+                    // Note commas cannot be put into the strings.  But, this should be a less likely case,
```

Comment:
What happens if there is a comma? Is that even possible to express?

Reply:
It's not possible to express at this moment, but I'm of the opinion for now, we can go with this, and then add escaping in a later version.